src/docs/background/13-how-payment-intents-work/index.md

```diff
@@ -0,0 +1,179 @@
+slug: payment-intents
+updated: 2019-06-19
+category: background
+ingress:
+  Overview of how Stripe PaymentIntents work with Sharetribe Flex, and how you can build your
+  transaction process with support for Strong Customer Authentication (SCA).
+published: true
+---
+## Introduction
+
+[PaymentIntents](https://stripe.com/docs/payments/payment-intents) are a
+mechanism provided by Stripe to track the lifecycle of customer checkout flow.
+In addition, PaymentIntents provide tools for [Strong Customer Authentication
+(SCA)](https://stripe.com/en-fi/payments/strong-customer-authentication) where
+required. Flex has built-in support for PaymentIntents and Strong Customer Authentication.
+
+In September 2019 [new European
+regulation](https://stripe.com/en-fi/payments/strong-customer-authentication)
+will begin requiring SCA for online payments from European customers. In order
+to conform to these regulatory changes, Stripe encourages users to [migrate or use
+ PaymentIntents](https://stripe.com/docs/strong-customer-authentication/migration)
+instead of directly creating Charges.
+
+In the future, PaymentIntents can be required in other countries outside
+Europe due to local regulatory changes but they can already be used
+outside Europe. And if needed, they can provide fraud prevention with
+things like [3D Secure Card
+Payments](https://stripe.com/docs/payments/3d-secure).
+
+This article will describe how PaymentIntents relate to Flex transaction
+processes and the general principles of implementing a checkout flow with the
+PaymentIntents.
+
+## Transaction process with PaymentIntents
+
+On high level, the payment flow with PaymentIntents has the following steps:
+
+1. Customer initiates (or transitions) a transaction with a transition
+   containing action that creates a PaymentIntent.
+2. Customer uses the PaymentIntent data to complete any steps
+   necessary to authenticate and authorize the payment.
+3. The transaction can proceed only after customer has authorized (if required)
+   the payment. The PaymentIntent is confirmed, resulting in a Charge being
+   preauthorized.
+4. Transaction flow continues as usual onwards.
+
+![Automatic PaymentIntent flow](./automatic_confirmation_flow.png)
+
+For technical implementation of PaymentIntents, Stripe offers two approaches -
+[manual or automatic confirmation
+flow](https://stripe.com/docs/payments/payment-intents#one-time-payments). We
+have implemented the support for the automatic flow. In practice, the Flex
+transaction engine models the automatic flow with two transitions. First transition
+creates the PaymentIntent (Step 1.) and second transition will validate and
+mark it confirmed in Flex (Step 3.). Between these steps, the automatic flow
+pushes the responsibility of authenticating, authorizing and confirming the
+payment in Stripe to the client application (Step 2.). More information on the
+Step 2. can be found in this [chapter](#required-changes-in-the-client).
```

Comment:
Are we calling the sections "chapters" or "sections" in the docs? "chapter" feels off to me, but if that's what we use elsewhere, no need to change.

Reply:
Section is better. Btw. We could utilize for these the suggestion mode:

```suggestion
Step 2. can be found in this [section](#required-changes-in-the-client).
```